libs/atqr/domain/src/lib/types/notification.types.ts

```diff
@@ -0,0 +1,32 @@
+import { ITemplateDataTypes } from '../services/notification/templateDataType.interface';
+
+export type TemplateDataTypes = {
+  [K in keyof ITemplateDataTypes]: ITemplateDataTypes[K];
+};
+
+export enum MessageTemplateId {
+  ChallengeCreated = 'ChallengeCreated',
+  ChallengeCanceled = 'ChallengeCanceled',
+  ChallengeOngoing = 'ChallengeOngoing',
+  ChallengeCompleted = 'ChallengeCompleted',
+  ChallengeFailed = 'ChallengeFailed',
+  YourOfficiationHaveBeenRequested = 'YourOfficiationHaveBeenRequested',
+  YourOfficiationRequestHaveBeenAccepted = 'YourOfficiationRequestHaveBeenAccepted',
+  YourOfficiationRequestHaveBeenRejected = 'YourOfficiationRequestHaveBeenRejected',
+  YourOfficiationRequestHaveBeenIgnored = 'YourOfficiationRequestHaveBeenIgnored',
+  YouHaveBeenChallenged = 'YouHaveBeenChallenged',
+  YourContenderAcceptedTheChallenge = 'YourContenderAcceptedTheChallenge',
+  YourContenderRejectedTheChallenge = 'YourContenderRejectedTheChallenge',
+  YourContenderIgnored = 'YourContenderIgnored',
+  YourContenderRemovedYouFromJudgeRole = 'YourContenderRemovedYouFromJudgeRole',
+  YourContenderCanceledTheChallenge = 'YourContenderCanceledTheChallenge',
+  YourContenderWereTheySuccessful = 'YourContenderWereTheySuccessful',
+  ReminderToOfficiate = 'ReminderToOfficiate',
+  YourJudgeDidNotAnswer = 'YourJudgeDidNotAnswer',
```

Comment:
These two are the ones who were changed. Sorry!

Check lines 35 and 37:
https://github.com/tokilabs/atqr/blob/ATQR-79-Create-Notifications-Value-Objects/libs/atqr/domain/src/lib/domain-spec/concept-set/Notifications.disl#L35